discordkit/utils/ClientTemplateGen.ts

```
@@ -0,0 +1,38 @@
+      const harmonix: Harmonix = {
+        client,
+        options,
+        commands: new Eris.Collection(),
+        slashCommands: new Eris.Collection(),
+        events: new Eris.Collection(),
+        startTime: new Date(),
+        manager: {} as any, // Manager initialization is deferred
+      };
+
+      return harmonix;
+    }),
```

Biome

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)